bin/_mocha

```diff
@@ -494,6 +496,13 @@ args.forEach(arg => {
     throw err;
   }
 
+  if (typeof newFiles !== 'undefined') {
```

Comment:
I don't think `newFiles` will be anything other than an `Array` of `string`s, will it?

Reply:
`newFiles` is the result of `utils.lookupFiles` which can return a string of arrays, a string, or `undefined`.